### PR TITLE
UNITY-8054 Sage REST: Create an endpoint that consumes the SKU and re…

### PR DESCRIPTION
**Pull request by bolsu**
…turns the GUID

UNITY-8054 Sage REST: Create an endpoint that consumes the SKU and returns the GUID

### CODE REVIEW

**Comment by bolsu** [comment]
@dkluever22 Hi Daria, just a heads up on the next REST update with the SAGE Epic, thanks!

File: _raml/apps/products/schema/post_product_find.schema.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      "type": "integer",
+      "description": "The number of products found in total."
+    },
+    "page": {
+      "type": "integer",
+      "description": "The page number on which the products appears."
+    },
+    "resultsPerPage": {
+      "type": "integer",
+      "description": "The number of products returned per page."
```
Comment (dkluever22) [inline _raml/apps/products/schema/post_product_find.schema.json:16]:
```suggestion
      "description": "The number of found products returned per page."
```

File: _raml/apps/products/examples/post_product_find_response.json [hunk 2]
```diff
@@ -0,0 +1,468 @@
+{
+  "results": 3,
+  "page": 1,
+  "resultsPerPage": 3,
+  "items": [
+    {
+      "productId": "54EF2E41-FCA8-E0E5-9323-0A0C05E66C2E",
+      "name": "Customised headphone",
```
Comment (dkluever22) [inline _raml/apps/products/examples/post_product_find_response.json:8]: I would suggest this changed wording cause these headphone are named this way in another example.
```suggestion
      "name": "Boom - Customised headphone",
```

**Review by dkluever22 — COMMENTED** [review]
@bolsu Please take a look at my wording suggestions :) Thanks in advance! 🙏

**Comment by bolsu** [comment]
Hi @dkluever22, I have merged your suggestions and done a rebase, thanks!

**Review by dkluever22 — APPROVED** [review]

**Pull request merged**